docs/source/blocks/block-style-wrapper.md

```diff
@@ -25,45 +25,68 @@ See below for an example.
 ## Enabling Style Wrapper in a block
 
 The wrapper is always present in the rendering of both the view and edit components.
-If you want to add the default set of styles, you need to enable them with the following flag:
+Once you add the `styles` field in your block schema, the wrapper will inject the class names derived from it into both the view and the edit components.
+
+### Using `styles` field in your block
+
+The wrapper builds the class names to be injected by looking up a field called `styles` in your block schema.
+As a schema it has the following signature, and it's normally placed in a `Styling` fieldset:
 
 ```js
-  // (in your block config object)
-  my_custom_block: {
-    // (more block settings)
-    enableStyling: true,
+const stylingSchema = {
+  fieldsets: [{
+    id: 'styling',
+    title: intl.formatMessage(messages.styling),
+    fields: ['styles'],
+  }],
+
+  properties: {
+    styles: {
+      widget: 'object',
+      title: intl.formatMessage(messages.styling),
+      schema: stylesSchema({
+        schema: defaultStyleSchema({ formData, intl }),
```

Comment:
@sneridagh this code snippet that is still added references **defaultStyleSchema**
which is now removed from this pull request.

Shouldn't the code snippet explain the empty styling fieldset from
https://github.com/plone/volto/pull/3794/files?short_path=43e361f#diff-8f34daa1151a8f71198bb80a59f0af841606e03e8e04c1c30a9825527283b279R296
which is then populated from 
https://github.com/plone/volto/pull/3794/files#diff-43e361f4bd5a52502dd6b2af742c7fb403abc31e7e8a61be75d76a6c49a733bfR70
?

Reply:
Fixed.